mlos_bench/setup.py

```diff
@@ -24,33 +25,44 @@
     warning(f"setuptools_scm failed to find git version, using version from _version.py: {e}")
 
 
-extra_requires = {
+extra_requires: Dict[str, List[str]] = {
```

Comment:
Makes the `chain()` composition easier to just standarize on dict of lists for all extra_requires.